modules/local/report.nf

```diff
@@ -671,10 +668,12 @@ def generate_qc_data(data, search_phrases, qc_species_tag){
     def quality_analysis = "QualityAnalysis"
     def shortest_token = get_shortest_token(search_phrases)
     def species_tag_location = 0
+    println "Data keys: ${data.keySet()}"
```

Comment:
Is this a line purely for debugging? Could you remove if it's no longer needed?

Reply:
it is! good catch :) thank you for finding that!

Reply:
fixed in: https://github.com/phac-nml/mikrokondo/pull/102/commits/aa889d85917670951646647930dcf7d1d2330ba9